### PR TITLE
Hide chat UI + bump app versions

### PR DESCRIPTION
**Pull request by dharit-tan**
### Description
Hide chats UI in profile screen + bump app versions for release

### Dragons

*Is there anything the reviewer should be on the lookout for? Are there any dangerous changes?*

### How Has This Been Tested?

Local ios against stage: 
Simulator Screen Shot - iPhone 14 Pro - 2023-03-09 at 11 54 03

### How will this change be monitored?

*For features that are critical or could fail silently please describe the monitoring/alerting being added.*

### Feature Flags ###

*Are all new features properly feature flagged? Describe added feature flags.*

### CODE REVIEW

**Review by sliptype — APPROVED** [review]
Do we not wanna use a feature flag for this?

**Comment by dharit-tan** [comment]
> Do we not wanna use a feature flag for this?

Honestly just wanna remove it for now to avoid any possible hiccups, will follow up with proper feature flagged version for next release.

**Comment by audius-infra** [comment]
It looks like there may be some changes to native mobile code, which requires triggering a full app release. Please follow the instructions here: https://www.notion.so/audiusproject/When-to-bump-app-version-2644a8f772364a4d91f44abcba44ce0b?pvs=4. cc @nicoback2 @sliptype

File: packages/mobile/ios/AudiusReactNative/Info.plist [hunk 1]
```diff
@@ -20,7 +20,7 @@
     <!-- 
       Make sure this is above the currently released iOS version in the app store if your changes touch native code.
     -->
-		<string>1.1.61</string>
+		<string>1.1.63</string>
```
Comment (nicoback2) [inline packages/mobile/ios/AudiusReactNative/Info.plist:23]:
```suggestion
		<string>1.1.62</string>
```
I think we didn't release 1.1.62 so you can use that. cc @sliptype

Comment (sliptype) [reply packages/mobile/ios/AudiusReactNative/Info.plist:23]: Yeah 1.1.62 will be better!

**Comment by audius-infra** [comment]
It looks like there may be some changes to native mobile code, which requires triggering a full app release. Please follow the instructions here: https://www.notion.so/audiusproject/When-to-bump-app-version-2644a8f772364a4d91f44abcba44ce0b?pvs=4. cc @nicoback2 @sliptype

**Review by nicoback2 — APPROVED** [review]

**Pull request merged**